x/pot/types/events.go

```diff
@@ -5,6 +5,7 @@ const (
 	EventTypeVolumeReport      = "volume_report"
 	EventTypeWithdraw          = "withdraw"
 	EventTypeFoundationDeposit = "foundation_deposit"
+	EventTypeSlashing          = "slashing"
```

Comment:
maybe slashing_resource_node is better, so we can have a different event for meta node slashing